packages/block-editor/src/components/rich-text/file-paste-handler.native.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
What's stopping us from reusing this for web? What's the reason for this `Number.MAX_SAFE_INTEGER` stuff? Could that be commented on?